hibernate-core/src/test/java/org/hibernate/test/cid/PurchaseRecordIdGenerator.java

```diff
@@ -8,14 +8,15 @@
 import java.io.Serializable;
 
 import org.hibernate.engine.spi.SessionImplementor;
+import org.hibernate.id.IdentifierGenerator;
 import org.hibernate.id.IdentityGenerator;
 
 /**
- * Simple {@link IdentityGenerator} implementation for testing composite-id.
+ * Simple {@link IdentifierGenerator} implementation for testing composite-id.
  * 
  * @author Jacob Robertson
  */
-public class PurchaseRecordIdGenerator extends IdentityGenerator {
+public class PurchaseRecordIdGenerator implements IdentifierGenerator {
```

Comment:
This is not required since `IdentityGenerator` implements `IdentifierGenerator` and it's a proof that the proposed changes don't affect generators that were created prior to this PR.